app/assets/stylesheets/components/_form.scss

```diff
@@ -1,3 +1,112 @@
+.wizard-form {
+  text-align: center;
+
+  form {
+    border-width: 0;
+    width: 70%
+  }
+
+  label {
+    font-size: 25px;
+    margin-bottom: 5px;
+    color: $light-gray;
+  }
+
+  .art-form-text {
+    margin-top: 50px;
+    height: 50px;
+  }
+
+  .textarea-holder {
+    margin-bottom: 200px;
+
+  }
+
+  input {
+    height: 50px;
+    font-size: 20px;
+    border-color: $gold;
+    background-color: $main-grey;
+    color: white;
```

Comment:
Can you please leave the color as lightgrey ? We don't white anywhere on the website, it would be strange to suddenly see it.